lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestSecurityToken.cs

```diff
@@ -0,0 +1,232 @@
+    {
+        private static readonly Logger Logger = Logger.GetLogger(typeof(TestSecurityToken));
+
+        private const string DefaultPortRangeStart = "2000";
+        private const string DefaultPortRangeCount = "20";
+
+        private const string Identifier1 = "TrustedApplicationTokenIdentifier";
+        private const string Identifier2 = "TrustedApplicationTokenIdentifier2";
+        private const string TokenKey1 = "TrustedApplication001";
+        private const string TokenKey2 = "TrustedApplication002";
+        private const string Password1 = "none";
+        private const string Password2 = "none";
+
+        /// <summary>
+        /// This is to test pass multiple tokens from client for Yarn application
+        /// </summary>
+        [Fact]
+        [Trait("Environment", "Yarn")]
+        [Trait("Priority", "1")]
+        [Trait("Description", "Run CLR Test on Yarn")]
+        public void TestSetMultipleSecurityToken()
+        {
+            TestRun(GetRuntimeConfigurationForMultipleTokens());
+        }
+
+        /// <summary>
+        /// This is to test write token in old approach for backward compatibility checking
+        /// </summary>
+        [Fact]
+        [Trait("Environment", "Yarn")]
+        [Trait("Priority", "1")]
+        [Trait("Description", "Run CLR Test on Yarn")]
+        public void TestSecurityTokenBackword()
+        {
+            TestRun(GetRuntimeConfigurationBackwardComp());
+        }
+
+        /// <summary>
+        /// This is to test passing one token from client for Yarn application
+        /// </summary>
+        [Fact]
+        [Trait("Environment", "Yarn")]
+        [Trait("Priority", "1")]
+        [Trait("Description", "Run CLR Test on Yarn")]
+        public void TestSetOneSecurityToken()
+        {
+            TestRun(GetRuntimeConfigurationForSingleToken());
+        }
+
+        /// <summary>
+        /// Test run for the runtime in the given injector.
+        /// </summary>
+        /// <param name="condig">runtime configuration.</param>
+        private void TestRun(IConfiguration condig)
+        {
+            IInjector injector = TangFactory.GetTang().NewInjector(condig);
+
+            var reefClient = injector.GetInstance<IREEFClient>();
+            var jobRequestBuilder = injector.GetInstance<JobRequestBuilder>();
+
+            var jobSubmission = jobRequestBuilder
+                .AddDriverConfiguration(GetDriverConfig())
+                .AddGlobalAssemblyForType(typeof(HelloDriver))
+                .SetJobIdentifier("MyTestJob")
+                .Build();
+
+            var result = reefClient.SubmitAndGetJobStatus(jobSubmission);
+            var state = PullFinalJobStatus(result);
+            Logger.Log(Level.Info, "Application final state : {0}.", state);
+            Assert.Equal(FinalState.SUCCEEDED, state);
+        }
+
+        /// <summary>
+        /// Use HelloDriver in the test
+        /// </summary>
+        /// <returns></returns>
```

Comment:
here and everywhere: I am OK if you don't document the private members, but please remove the bolerplate tags that you don't use. e.g. here either add some meaningful description to the `<returns>` section, or remove it completely.